### PR TITLE
Feature/uifixes

### PR DESCRIPTION
**Pull request by CBarrett2**

### CODE REVIEW

File: src/assets/js/main.js [hunk 1]
```diff
@@ -1,38 +1,202 @@
...
+    var rsiHeight = height / 2;
+    var navHeight = height / 3;
 
-    var svg = d3.select('#chart-example')
-        .append('svg')
+
+    // Set SVGs
+    var container = d3.select('#chart-example');
+    var svgMain = container.select('svg.main')
+        .attr('viewBox', function() { return '0 0 ' + width + ' ' + height; })
```
Comment (jleft) [inline src/assets/js/main.js]: Do we need `viewBox`?

File: src/assets/js/main.js [hunk 2]
```diff
@@ -1,38 +1,202 @@
...
+        .xTicks(0);
+
+    // Create and apply the Moving Average
+    var movingAverage = fc.indicator.algorithm.movingAverage();
+
+    // Create a line that renders the result
+    var ma = fc.series.line()
+        .yValue(function(d) { return d.movingAverage; });
+
+    var multi = fc.series.multi().series([candlestick, gridlines, ma]);
```
Comment (jleft) [inline src/assets/js/main.js]: Can we pass in gridlines first, so they're rendered under the candlesticks and ma?

File: src/assets/js/main.js [hunk 3]
```diff
@@ -1,38 +1,202 @@
...
+
+    function zoomCall(zoom, data, scale) {
```
Comment (jleft) [inline src/assets/js/main.js]: I'm still finding some slightly peculiar behaviour with the zoom - we can raise bugs against this.

File: src/assets/js/main.js [hunk 4]
```diff
@@ -1,38 +1,202 @@
...
+    var rsiScale = d3.scale.linear()
+        .domain([0, 100])
+        .range([rsiHeight, 0]);
+    var rsiAlgorithm = fc.indicator.algorithm.relativeStrengthIndex();
+
+    var rsi = fc.indicator.renderer.relativeStrengthIndex()
+        .xScale(timeSeries.xScale())
+        .yScale(rsiScale);
+
+    var rsiChart = function(selection) {
```
Comment (jleft) [inline src/assets/js/main.js]: I think it's worth considering if this should have its own x-axis, happy to leave as-is for now though.

File: src/assets/js/main.js [hunk 5]
```diff
@@ -1,38 +1,202 @@
...
+                }
+            });
 
-    svg.datum(data)
-        .call(chart);
+        // Allow to zoom using mouse, but disable panning
+        var zoom = d3.behavior.zoom()
+            .x(timeSeries.xScale())
+            .on('zoom', function() {
+                if (zoom.scale() === 1) {
```
Comment (jleft) [inline src/assets/js/main.js]: Should we be making use of `zoomCall()` here, to limit the zoom?

**Comment by CBarrett2** [comment]
Removed the extra features for now, still working on a fix but thought at least without them it might be able to get pulled

**Comment by jleft** [comment]
> Removed the extra features for now, still working on a fix but thought at least without them it might be able to get pulled

Great, thank you :)

Please can we add Bootstrap as a dependency, then this should be good to go:
```sh
npm install bootstrap --save
```

**Pull request closed without merging**